eng/pipelines/libraries/run-test-job.yml

```diff
@@ -7,6 +7,8 @@ parameters:
   isOfficialBuild: false
   liveRuntimeBuildConfig: ''
   runtimeFlavor: 'coreclr'
+  runtimeDisplayName: 'coreclr'
+  interpreter: ''
```

Comment:
As we discussed offline, maybe it makes sense to use the already known `runtimeMode` as a switch instead of adding a boolean flag: https://github.com/dotnet/runtime/blob/master/eng/pipelines/runtime.yml#L746